### ISSUE TITLE
Add possibility to disable notifications from bloc extension for VSCode

### ISSUE THREAD

**Issue by minikin** [felangel/bloc#1962]
Bloc extension notifications for VSCode are bothersome.

<img width="615" alt="Screenshot 2020-11-24 at 14 54 47" src="https://user-images.githubusercontent.com/4456735/100103830-b351f700-2e65-11eb-9a7f-fb00f7d047f8.png">

**Describe the solution you'd like**
As a user, I want to have the possibility to disable or ignore notifications from the bloc extension for VSCode in settings.

**Describe alternatives you've considered**
Deleting bloc extension for VSCode.

**Comment by felangel** [comment]
Hi @minikin 👋 
Thanks for opening an issue!

I'll add a setting to the extension to disable the upgrade notifications in the next version 👍 
Sorry for the inconvenience!

**Comment by minikin** [comment]
Thank you very much @felangel 👍

**Comment by felangel** [comment]
@minikin this was addressed in #1964 and has been included in the v5.5.0 release of the vscode extension 🎉

**Comment by NatoBoram** [comment]
This is the setting :

```jsonc
// Whether to check if you are using the latest package versions at startup.
"bloc.checkForUpdates": false
```
